release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1462,6 +1598,10 @@ This script removes unauthenticated subjects from the following cluster role bin
 // TODO: This known issue should carry forward to 4.9 and beyond!
 * The `oc annotate` command does not work for LDAP group names that contain an equal sign (`=`), because the command uses the equal sign as a delimiter between the annotation name and value. As a workaround, use `oc patch` or `oc edit` to add the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917280[*BZ#1917280*])
 
+* In the monitoring stack, if you have enabled and deployed a dedicated Alertmanager instance for user-defined alerts, you cannot silence alerts in the *Developer* perspective in the {product-title} web console.
+No workaround exists for this issue.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2100860[*BZ#2100860*])
+
```

Comment:
also worth mentioning https://bugzilla.redhat.com/show_bug.cgi?id=2103127

```
* On a newly installed 4.11 cluster, the platform alerts will not have the `openshift_io_alert_source="platform"` label. This issue doesn't affect clusters upgraded from a previous minor version.
 No workaround exists for this issue.
```

Reply:
Added the following known issue:

```
* On a newly installed {product-title} {product-version} cluster, platform monitoring alerts do not have the `openshift_io_alert_source="platform"` label. 
This issue does not affect clusters upgraded from a previous minor version.
No workaround exists for this issue.
(link:https://bugzilla.redhat.com/show_bug.cgi?id=2103127[*BZ#2103127])
```